content/en/building/targets/targets.md

```diff
@@ -1,16 +1,16 @@
 ---
```

Comment:
I would name this file `targets-widgets.md` to make it less confusing. What do you think?

Reply:
Yes, let me do that.

Reply:
Is it better as target-widgets or targets-widgets?

Reply:
I have kept it as targets-widgets

Reply:
Thanks @andrablaj for helping me ponder on the small details. I really appreciate it.